pom.xml

```diff
@@ -25,7 +25,7 @@
     </parent>
 
     <artifactId>emitter</artifactId>
-    <version>0.6.0-rc2-SNAPSHOT</version>
+    <version>0.6.0-rc3-SNAPSHOT</version>
```

Comment:
The project is in fact on RC stage, so I don't see the point. OTOH want this to be in master already, to create other PRs based on this branch. And I don't want to create commits into this `refactor-http-and-parametrized-emitter-configs` feature branch, because when in the end it will be merged into master, it will contain a lot of unrelated changes.